submodules/LIB/scripts/submodule_utils.py

```diff
@@ -571,9 +571,9 @@ def get_reserved_signals(signal_list):
 def get_reserved_signal_connection(signal_name, instace_name, swreg_filename):
     signal_connection = reserved_signals[signal_name]
     return re.sub(
-        "\/\*<InstanceName>\*\/",
+        r"\/\*<InstanceName>\*\/",
         instace_name,
-        re.sub("\/\*<SwregFilename>\*\/", swreg_filename, signal_connection),
+        re.sub(r"\/\*<SwregFilename>\*\/", swreg_filename, signal_connection),
```

Comment:
To give more details:
According to [python3 documentation](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals), an occurence of `\[character]` in a string is considered an escape sequence; except for strings with prefixes `r""` or `R""`.
Without `r""` prefix, the string: `"\/\*<InstanceName>\*\/"` tries to evaluate the `\/` as an escape sequence and can give a warning dependent on the python version:
```
Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, 
i.e., the backslash is left in the result. (This behavior is useful when debugging: 
if an escape sequence is mistyped, the resulting output is more easily recognized as broken.)
It is also important to note that the escape sequences only recognized in string literals fall 
into the category of unrecognized escapes for bytes literals.

Changed in version 3.6: Unrecognized escape sequences produce 
a [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning).

Changed in version 3.12: Unrecognized escape sequences produce 
a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning). 
In a future Python version they will be eventually 
a [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError).
``` 
Running in python3.10:
```bash
$ python3
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a_tring = "\/\*<InstanceName>\*\/"
>>> 
```
Running in python3.12:
```bash
$ python3
Python 3.12.1 (main, Dec 19 2023, 12:55:07) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a_tring = "\/\*<InstanceName>\*\/"
<stdin>:1: SyntaxWarning: invalid escape sequence '\/'
>>>
```

**TLDR:** different python versions give different warnings regarding string escape sequences in strings. 
- Since 3.6 gives Deprecation Warning; 
- Since 3.12 gives SyntaxWarning. 
- Planned to be SyntaxError in some future version.
I only saw this, since I use python3.12, but iobundle machines run 3.10. We can ignore these warnings for now